open-metadata-implementation/access-services/subject-area/subject-area-server/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/server/services/SubjectAreaRESTServicesInstance.java

```diff
@@ -36,7 +36,7 @@
     private static final String className = SubjectAreaRESTServicesInstance.class.getName();
 
     public static final String PAGE_OFFSET_DEFAULT_VALUE = "0";
-    public static final String PAGE_SIZE_DEFAULT_VALUE = "100";
+    public static final String PAGE_SIZE_DEFAULT_VALUE = "0";
     public static final String TRUE_DEFAULT_VALUE = "true";
     public static final String FALSE_DEFAULT_VALUE = "false";
```

Comment:
I am not sure this constant adds any value - I suggest leaving as false ;unless you want to include the paramter name in the constant - I am easy either way.